app/packages/looker/src/worker/canvas-decoder.ts

```diff
@@ -0,0 +1,36 @@
+import { OverlayMask } from "../numpy";
+
+/**
+ * Decodes a given image source into an OverlayMask using an OffscreenCanvas
+ */
+export const decodeWithCanvas = async (blob: ImageBitmapSource) => {
+  const imageBitmap = await createImageBitmap(blob);
```

Comment:
I wonder if we could use `ImageBitmap` in the `Overlay` classes instead of `ImageDate`. It seems like that could be a performance improvement.

Just thinking out loud. Nothing actionable in this PR

Reply:
I like the idea. Let's explore in another thread.